apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -364,19 +369,13 @@ export const PageTemplatesPlugin = compose(
 			postContentBlock: select( 'core/editor' )
 				.getBlocks()
 				.find( block => block.name === 'a8c/post-content' ),
-			isWelcomeGuideActive: select( 'core/edit-post' ).isFeatureActive( 'welcomeGuide' ),
+			isWelcomeGuideActive: select( 'core/edit-post' ).isFeatureActive( 'welcomeGuide' ), // Gutenberg 7.2.0 and higher
+			areTipsEnabled: select( 'core/nux' ).areTipsEnabled
```

Comment:
This can break with a clean local storage:
```
Error: An error occurred while running 'mapSelect': Cannot read property 'areTipsEnabled' of null
The error may be correlated with this previous error:
TypeError: Cannot read property 'areTipsEnabled' of null
    at http://localhost:8889/wp-content/plugins/full-site-editing-plugin/starter-page-templates/dist/starter-page-templates.js?ver=1577712450:6:19228
    at Object.mapSelect [as current] (http://localhost:8889/wp-content/plugins/gutenberg/build/data/index.js?ver=c595d6e0617e6d3f90229798e56a7e1d:2538:16)
    at onStoreChange (http://localhost:8889/wp-content/plugins/gutenberg/build/data/index.js?ver=c595d6e0617e6d3f90229798e56a7e1d:2250:46)
    at http://localhost:8889/wp-content/plugins/gutenberg/build/data/index.js?ver=c595d6e0617e6d3f90229798e56a7e1d:2277:9
    at http://localhost:8889/wp-content/plugins/gutenberg/build/data/index.js?ver=c595d6e0617e6d3f90229798e56a7e1d:4128:14
    at Array.forEach (<anonymous>)
    at globalListener (http://localhost:8889/wp-content/plugins/gutenberg/build/data/index.js?ver=c595d6e0617e6d3f90229798e56a7e1d:4127:15)
    at http://localhost:8889/wp-content/plugins/gutenberg/build/data/index.js?ver=c595d6e0617e6d3f90229798e56a7e1d:2997:9
    at Object.dispatch (http://localhost:8889/wp-content/plugins/gutenberg/build/data/index.js?ver=c595d6e0617e6d3f90229798e56a7e1d:1089:7)
    at e (<anonymous>:1:40553)

Original stack trace:
    at useSelect (http://localhost:8889/wp-content/plugins/gutenberg/build/data/index.js?ver=c595d6e0617e6d3f90229798e56a7e1d:2230:13)
    at http://localhost:8889/wp-content/plugins/gutenberg/build/data/index.js?ver=c595d6e0617e6d3f90229798e56a7e1d:2541:83
    at renderWithHooks (http://localhost:8889/wp-content/plugins/gutenberg/vendor/react-dom.8b3dda97.js?ver=16.9.0:15246:18)
    at updateFunctionComponent (http://localhost:8889/wp-content/plugins/gutenberg/vendor/react-dom.8b3dda97.js?ver=16.9.0:17063:20)
    at beginWork$1 (http://localhost:8889/wp-content/plugins/gutenberg/vendor/react-dom.8b3dda97.js?ver=16.9.0:18636:16)
    at HTMLUnknownElement.callCallback (http://localhost:8889/wp-content/plugins/gutenberg/vendor/react-dom.8b3dda97.js?ver=16.9.0:341:14)
    at Object.invokeGuardedCallbackDev (http://localhost:8889/wp-content/plugins/gutenberg/vendor/react-dom.8b3dda97.js?ver=16.9.0:391:16)
    at invokeGuardedCallback (http://localhost:8889/wp-content/plugins/gutenberg/vendor/react-dom.8b3dda97.js?ver=16.9.0:448:31)
    at beginWork$$1 (http://localhost:8889/wp-content/plugins/gutenberg/vendor/react-dom.8b3dda97.js?ver=16.9.0:23355:7)
    at performUnitOfWork (http://localhost:8889/wp-content/plugins/gutenberg/vendor/react-dom.8b3dda97.js?ver=16.9.0:22346:12)
```

It should be enough to just check if the `core/nux` package exists:

```js
areTipsEnabled: select( 'core/nux' ) ? select( 'core/nux' ).areTipsEnabled() : false
```

Reply:
You're right! I assumed this was behaving like `dispatch` (`dispatch( 'core/nux' )` returns `{}` when the nux package is not imported). Fixed in 0909d6b